hcl/scanner/scanner.go

```diff
@@ -224,7 +224,11 @@ func (s *Scanner) scanComment(ch rune) {
 	// single line comments
 	if ch == '#' || (ch == '/' && s.peek() != '*') {
 		ch = s.next()
-		for ch != '\n' && ch >= 0 && ch != eof {
+		for ch != '\n' && ch != '\r' && ch >= 0 && ch != eof {
+			if ch == '\r' && s.peek() == '\n' {
```

Comment:
The question is do we want the `\r` to be included in the comment or treated as part of the line break? Unlike HEREDOCs I'd say they should be part of the line break for comments (though unless we're going to process comments it doesn't make much difference).